cluster-service/deploy/openshift-templates/arohcp-service-template.yml

```diff
@@ -222,8 +222,35 @@ parameters:
 - name: CS_SERVICE_PRINCIPAL_CREDS_BASE64
   description: Base64 encoded service principal credentials for CS. This is temporary for P1
   required: true
+- name: KEYVAULT_NAME
+  description: The name of the Key Vault where the first party credentials application are stored.
+  value: "service-kv-aro-hcp-dev"
+- name: TENANT_ID
+  description: The Tenant ID of Key Vault.
+  value: "64dc69e4-d083-49fc-9569-ebece1dd1408"
+
 objects:
 
+- apiVersion: secrets-store.csi.x-k8s.io/v1
+  kind: SecretProviderClass
+  metadata:
+    name: cs-keyvault
+    namespace: ${NAMESPACE}
+  spec:
+    parameters:
+      clientID: ${AZURE_CS_MI_CLIENT_ID}
```

Comment:
@geoberle should this be CS' identity used here? Shouldn't we have a dedicated identity for secrets CSI storage driver to mount the secret into the pod, rather than give it broad permissions CS has?

Reply:
good point. i would not hold off this PR from being merged because of this though. this is something we should do for other services as well, e.g. maestro-server

i've created some cards to track this work
https://issues.redhat.com/browse/ARO-10155

Reply:
Once those identities are created they can be changed here. Resolving the conversation.